### PR TITLE
Documentation for start.kubermatic.io

### PR DESCRIPTION
**Pull request by michalvanco**
Includes a proposal of the documenation for start.kubermatic.io.

Documentation is a h1 level under KKP, we are still waiting for some updated schemas.

Content is added only under `master` version (as we will iterate on top of it) and we can potentially remove that from the links in left navigation bar if needed.

Fixes kubermatic/startio#33.

### CODE REVIEW

File: content/kubermatic/master/startio/cheat_sheets/_index.en.md
Comment (xmudrii) [inline content/kubermatic/master/startio/cheat_sheets/_index.en.md:4]: Tiny nit: we should add a short introductory text/description for this section. We already have that for other sections.

File: content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md
Comment (xmudrii) [inline content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md:9]: I think we should use level two headers (`##`) instead of level three. Also, I'd use some notation to highlight that this is the first, second, third step, for example:

> 1. Retrieve Terraform State
> 2. Prepare SSH Agent with SSH Key
> 3. Use KubeOne to retrieve kubeconfig

File: content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md
Comment (xmudrii) [inline content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md:7]: Tiny nit: I think we can replace `:` with `.`.

```suggestion
You can either login to created EC2 instance and use the `/etc/kubernetes/admin.conf` or if you want to
have access to your Master Kubernetes cluster locally, follow these steps.
```

File: content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md
Comment (xmudrii) [inline content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md:2]: This guide should have a prerequisite section specifying that you need Terraform and KubeOne locally with links to relevant installation guides. I know that we already specify tools used in the pipeline in the concepts documents, but it might be useful to specify which tools you need locally and how to get them for each guide.

File: content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md
Comment (xmudrii) [inline content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md:23]: It might be useful to specify that this is a KubeOne requirement and link to the appropriate KubeOne doc: https://docs.kubermatic.com/kubeone/master/guides/ssh/

File: content/kubermatic/master/startio/cheat_sheets/work_with_secrets/_index.en.md
Comment (xmudrii) [inline content/kubermatic/master/startio/cheat_sheets/work_with_secrets/_index.en.md:6]: It would be useful to specify which SOPS versions we use in the pipeline, so folks can get the same version locally.

File: content/kubermatic/master/startio/guides/setup_repository/_index.en.md
Comment (xmudrii) [inline content/kubermatic/master/startio/guides/setup_repository/_index.en.md:75]: Which details should the user/reader see? If we just link directly to the landing page, the user might be confused about what they should see and read. It might be useful to directly link to some useful tutorials/guides, such as creating the user cluster, etc.

**Review by xmudrii — COMMENTED** [review]
Overall, the docs and the structure are really good. This is definitely a great starting point. We could eventually extend some sections and add additional descriptions, but I would get started with this and iterate in the future.

For now, I left some comments that we should consider addressing before publishing the docs. The most of comments should be minor and easy to address.

File: content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md
Comment (xmudrii) [inline content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md:11]: Can we use a Hugo reference here?

**Review by xmudrii — COMMENTED** [review]
One minor nit, otherwise LGTM. Great job!

**Comment by michalvanco** [comment]
I am waiting for updated images and then I will squash the commits, thanks @xmudrii!

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve
/hold
if you want the final approval from @shaase-ctrl

File: content/kubermatic/master/startio/_index.en.md
Comment (shaase-ctrl) [inline content/kubermatic/master/startio/_index.en.md:11]: A web-application that bootstraps the installation of your Kubermatic Kubernetes Platform and additional tools within a feew minutes in an automated way with applied GitOps principles.

File: content/kubermatic/master/startio/guides/setup_repository/_index.en.md
Comment (shaase-ctrl) [inline content/kubermatic/master/startio/guides/setup_repository/_index.en.md:7]: The goal of the setup

File: content/kubermatic/master/startio/guides/wizard/_index.en.md
Comment (shaase-ctrl) [inline content/kubermatic/master/startio/guides/wizard/_index.en.md:26]: How many nodes do we configure? 3?

File: content/kubermatic/master/startio/_index.en.md
Comment (shaase-ctrl) [inline content/kubermatic/master/startio/_index.en.md:25]: We should not mention the open sourcing yet, we may want to keep it closed depending of the outcome and the possibility to add a shop for buying EE.

File: content/kubermatic/master/startio/guides/wizard/_index.en.md
Comment (shaase-ctrl) [inline content/kubermatic/master/startio/guides/wizard/_index.en.md:49]: Should this be optional, or do we recommend this?

File: content/kubermatic/master/startio/guides/wizard/_index.en.md
Comment (michalvanco) [reply content/kubermatic/master/startio/guides/wizard/_index.en.md:26]: yes, 3 by default... I will also check whether it's regional or one-region setup... and I would also consider deploying autoscaler addon OOB, wdyt?

File: content/kubermatic/master/startio/guides/wizard/_index.en.md
Comment (michalvanco) [reply content/kubermatic/master/startio/guides/wizard/_index.en.md]: it's optional... and of course, we recommend it ;)

File: content/kubermatic/master/startio/_index.en.md
Comment (michalvanco) [reply content/kubermatic/master/startio/_index.en.md]: removed

**Review by xmudrii — APPROVED** [review]

**Pull request merged**
